src/es/components/molecules/teaser/Teaser.js

```diff
@@ -18,6 +18,23 @@ export default class KsTeaser extends Teaser {
       importMetaUrl: import.meta.url,
       ...options
     }, ...args)
+
+    if (this.getAttribute('namespace') === 'teaser-text-image-') {
```

Comment:
You could have extended the prototype class Hover: https://github.com/mits-gossau/web-components-toolbox/blob/master/src/es/components/prototypes/Hover.js to accomplish the same result.